packages/workbox-runtime-caching/test/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  globals: {
```

Comment:
Did you mean to add this? I thought you wanted to remove all but the base eslintrc file.

I tend to prefer declaring globals via inline eslint comments on a per-file basis (and the only globals declared in eslintrc files are globals that apply generally to the entire project).

That way when you see a variable in a file, it's very clear that the intent is to make it global, and you don't have to look up the directory structure for eslintrc files. WDYT?